docs/Users/Tutorials/Tutorial.rst

```diff
@@ -55,6 +55,11 @@ kind of analysis with the ``--bears`` argument:
     cd coala-tutorial
     coala --files=src/\*.c --bears=SpaceConsistencyBear --save
 
+.. note::
+
+    You can use comma seperated values to specify more than one item in
+    arguments! E.g. ``SpaceConsistencyBear,PEP8Bear``.
```

Comment:
Yeah, it's fairly obvious, but I think it will be worth mentioning (it could be costly to them to not run a bear)